developer/src/inst/build.sh

```diff
@@ -242,25 +242,24 @@ function make-installer() {
   #
 }
 
-# TODO: rename this to keyman-developer-cli-$Version.zip
-KMC_ZIP="$DEVELOPER_ROOT/release/$KEYMAN_VERSION/kmcomp-$KEYMAN_VERSION.zip"
-
 function make-kmc-install-zip() {
   builder_heading make-kmc-install-zip
 
   copy-schemas
-  cd "$DEVELOPER_ROOT/bin"
+  cd "${DEVELOPER_ROOT}/bin"
+
+  # TODO: rename this to keyman-developer-cli-$Version.zip
+  local KMCOMP_ZIP="${DEVELOPER_ROOT}/release/${KEYMAN_VERSION}/kmcomp-${KEYMAN_VERSION}.zip"
+
+  # shellcheck disable=SC2154
+  local COMPRESS_CMD="${SEVENZ_HOME}/7z"
 
-  wzzip -bd -bb0 "$KMC_ZIP" \
+  "${COMPRESS_CMD}" a -bd -bb0 "${KMCOMP_ZIP}" \
```

Comment:
Maybe a TODO on a separate PR to refactor this to use 

https://github.com/keymanapp/keyman/blob/e366117dee2d3287bc130a73ac306b9f31b4a9eb/resources/build/zip.inc.sh#L18

Reply:
I created #14332

Reply:
Done in #14334